test/cruds/move-columns.spec.ts

```diff
@@ -239,6 +239,17 @@ describe('Move columns', () => {
 
     expect(engine.lazilyTransformingAstService.version()).toEqual(version + 1)
   })
+
+  it('leaves the engine in a valid state so other operations are possible afterwards', () => {
```

Comment:
Maybe it would be a good idea to add more test cases:
- similar to this one in `move-rows.spec.ts`, `set-column-order.spec.ts`, `move-cells.spec.ts`
- some test case where there is a range in use

Reply:
Added